Makefile

```diff
@@ -25,7 +25,7 @@ setup-env:
 	fi
 
 lint: setup-env
-	golangci-lint run
+	golangci-lint run --max-same-issues 50 -E goimports  ./...
```

Comment:
I think this is supposed to be in the .golangci.yaml

Reply:
whoops. im really not very good at this

Reply:
Looks like `-E` is additive: https://golangci-lint.run/usage/configuration/

> Slice options (e.g. list of enabled/disabled linters) are combined from the command-line and config file.

but I do think it would be nice to keep all the config in one place